src/compressed_tensors/compressors/pack_quantized.py

```diff
@@ -101,7 +111,10 @@ def compress(
         return compressed_dict
 
     def decompress(
-        self, path_to_model_or_tensors: str, device: str = "cpu"
+        self,
+        path_to_model_or_tensors: str,
+        names_to_scheme: Dict[str, QuantizationArgs],
```

Comment:
Could we make this the same arg name as the compress method? I do like this name better though, we could change it in the compress function